tests/integration_tests/functional/test_initrd.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for initd."""
+
+import pytest
+from host_tools.network import SSHConnection
+from framework.microvm import Serial
+
+INITRD_FILESYSTEM = "rootfs"
+
+
+@pytest.mark.xfail(raises=ConnectionError)
+def test_microvm_initrd_with_ssh(
```

Comment:
I see that the initrd provided does not add support for networking, the eth0 interface cannot be found by the networking service upon guest start.  So this could explain why this test is failing. However, as far as my concern, this test could be deleted since initd support should work without network support.

Reply:
Deleted